base.mk

```diff
@@ -122,7 +122,7 @@ CFLAGS += -DITA_NO_ASSERT
 $(info Asserts will be switched off in debug mode due to code size limitation)
 endif
 else ifeq ($(BUILD), release)
-CFLAGS += -Os -fomit-frame-pointer
+CFLAGS += -Os -fomit-frame-pointer -flto
```

Comment:
Yes @jeez, exactly, in release 1.3 we will have a new experimental ``BUILD=lto`` build option. The plan is to have it fully validated for 1.4.

@keton, thanks a lot for your contribution. I hope it is fine for you to have this merged after 1.3.